### DISCUSSION TITLE
想问一下不同版本的MC，所需选择的JAVE版本分别是那些

### DISCUSSION THREAD

**Discussion by TXSQ2009** [Meloong-Git/PCL#1614] [提问] [answered]
想问一下不同版本的MC，所需选择的JAVE版本分别是那些

**Comment by hejiehao** [comment]
mc版本≤1.16.5：java8
mc版本≥1.17：java17
如果mod或整合包让你使用特定版本就使用他们让你使用的特定版本

**Comment by WForst-Breeze** [comment, marked as answer]
按照 Mojang 的说法是这样的：

1.6.1 以前所有版本：Java 5
1.6.1-1.12（不含）：Java 6
1.12-1.17（不含）：Java 8
1.17 及其下属版本：Java 16
1.18 至今：Java 17

但是实际使用时一般只需要准备 Java 8 和 Java 17 即可

**Comment by so1ve** [comment]
直接上scoop或者winget，把jdk全部装过去就完事了

**Comment by Pigeon0v0** [comment]
个人测试情况来看，甚至原版 1.0 都能用 Java 17 打开...

**Comment by Super-XiaoNian** [comment]
摘自Minecraft 中文Wiki
从[1.12](https://minecraft.fandom.com/zh/wiki/Java%E7%89%881.12)（[17w13a](https://minecraft.fandom.com/zh/wiki/17w13a)）开始，运行Minecraft的最低要求是Java 8。如果玩家不知道他们的电脑是否安装了Java 8，那么安装程序默认会自动安装并使用支持Minecraft运行的Java。
从[1.17](https://minecraft.fandom.com/zh/wiki/Java%E7%89%881.17)（[21w19a](https://minecraft.fandom.com/zh/wiki/21w19a)）开始，运行Minecraft的最低要求是Java 16。
从[1.18](https://minecraft.fandom.com/zh/wiki/Java%E7%89%881.18)（[1.18-pre2](https://minecraft.fandom.com/zh/wiki/Java%E7%89%881.18-pre2)）开始，运行Minecraft的最低要求是Java 17。

**Reply by Super-XiaoNian** [reply]
目前经验来看
1.7.10可以使用Java 7，也可以用Java 8 ，以下只能用7
1.8以上只能用Java 8
